### ISSUE TITLE
Consider using org.godotengine.godot.BaseApp

### ISSUE THREAD

**Issue by hexagonrecursion** [flathub/in.p1x.TanksOfFreedom#4]
It looks like someone created [a base app for games that use the Godot framework](https://github.com/flathub/org.godotengine.godot.BaseApp). You should consider using it.

[What is a base app?](https://docs.flatpak.org/en/latest/dependencies.html#base-apps)

**Comment by barthalion** [comment]
I don't think this package is maintained these days. If it's something you're interested in, I can grant you write access to the repo so you are nto blocked on me.

**Comment by hexagonrecursion** [comment]
Sorry. I have enough on my plate for now. Thank you for offering.

**Comment by czlowiekimadlo** [comment]
This game uses Godot version 2.1.x, while base app only offers versions 3.x, so it would not work anyway.

BTW there is new version of the game - 0.7.1-beta - with a couple of fixes and an SSL cert update. This change could be bundled up with Godot version bump to 2.1.6
